tests/unit_test/utility_test.py

```
@@ -2806,21 +2863,15 @@ def test_special_match(self):
-
-    def test_contains_special_characters_with_ampersand(self):
-        assert Utility.contains_special_characters(strg="Tom & Jerry") is True
-
+    @pytest.mark.parametrize("input_str, expected", [
+        ("Testing@123", True),
+        ("Testing 123", False),
+        ("Testing-123", False),
+        ("Testing_123", False),
+        ("Tom & Jerry", True)
+    ])
+    def test_special_match(self, input_str, expected):
```

Ruff (0.8.2)

Redefinition of unused `test_special_match` from line 2860

(F811)